app/build.gradle

```diff
@@ -22,7 +22,7 @@ android {
 
     defaultConfig {
         applicationId "io.intrepid.contest"
-        minSdkVersion 17
+        minSdkVersion 19
```

Comment:
I thought I would need it for cardview elevation, but I don't think I will. reverting!